test/cookbooks/sensu_test/recipes/default.rb

```diff
@@ -29,3 +29,40 @@
     sha512 property['checksum']
   end
 end
+
+sensu_filter 'production_filter' do
+  filter_action 'allow'
+  statements [
+    "event.Entity.Environment == 'production'",
+  ]
+end
+
+sensu_filter 'development_filter' do
+  filter_action 'deny'
+  statements [
+    "event.Entity.Environment == 'production'",
```

Comment:
`s/production/development`

Reply:
Actually that one confused me at first too, but it's to discard event data that has the 'production' attribute. It's straight from here: https://docs.sensu.io/sensu-core/2.0/reference/filters/#handling-non-production-events

Reply:
oh I did not realize the `deny` above that makes perfect sense now though I'd argue it would need to check a substring as many places use env names such as `production-west` and `production-east` but that's fine for an example.